### PR TITLE
Sentient Outpost

### PR DESCRIPTION
**Pull request by EricSihaoLin**
**Summary:**

This PR will potentially do the following:

Fix safari favicon
Introduce new beta component tracking Sentient Outpost

---
**Fixes issue (include link):**

---
**Mockups, screenshots, evidence:**

image

### CODE REVIEW

File: src/components/panels/SentientOutpostsPanel.vue [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  name: 'SentientOutpostsPanel',
+  props: ['sentientOutposts'],
+  computed: {
+    headertext() {
+      return 'Sentient Outposts (Beta)';
+    },
+  },
+  components: {
+    HubPanelWrap,
+    TimeBadge,
```
Comment (EricSihaoLin) [inline src/components/panels/SentientOutpostsPanel.vue]: Codacy Issue found: [The "TimeBadge" component has been registered but not used. (vue/no-unused-components)](https://app.codacy.com/app/WFCD/warframe-hub/pullRequest?prid=4702431)

**Comment by EricSihaoLin** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/WFCD/warframe-hub/pullRequest?prid=4702431&bid=15711580)

**Comment by EricSihaoLin** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/WFCD/warframe-hub/pullRequest?prid=4702431&bid=15711580)

File: src/components/panels/SentientOutpostsPanel.vue [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
```
Comment (EricSihaoLin) [inline src/components/panels/SentientOutpostsPanel.vue:41]: Codacy Issue found: [The "TimeBadge" component has been registered but not used. (vue/no-unused-components)](https://app.codacy.com/app/WFCD/warframe-hub/pullRequest?prid=4702431)

**Review by TobiTenno — APPROVED** [review]
lgtm

**Pull request merged**